content/shmem_quiet.tex

```diff
@@ -1,8 +1,6 @@
 \apisummary{
-    Waits for completion of all outstanding memory store, blocking 
-    \PUT{}, \ac{AMO}, and \OPR{put-with-signal}, as well as 
-    nonblocking \PUT{}, \OPR{put-with-signal}, \GET{}, and \ac{AMO} routines 
-    to symmetric data objects issued by a \ac{PE}.
+    Waits for completion of all outstanding operations on symmetric data objects
```

Comment:
```suggestion
    Waits for completion of outstanding operations on symmetric data objects
```

Suggest removing "all", as the outstanding operations are limited in scope to a (implicit or explicit) context.